packages/polyfills/console.js

```diff
@@ -274,7 +274,7 @@ const inspect = (function () {
       }
       name = JSON.stringify('' + key);
       if (name.match(/^"([a-zA-Z_][a-zA-Z_0-9]*)"$/)) {
-        name = name.substr(1, name.length - 2);
+        name = name.slice(1, name.length - 1);
```

Comment:
There are no tests in place for utility/helper functions.
I forgot to modify the second param and all tests passed in CI, such changes could be easily missed by anyone.

For each utility/helper function there should be test in-place.
cc: @cortinico / @jacdebug WDYT? Should we work on increasing the test-coverage for utility/helper functions?

Reply:
If there is no code coverage, testing the console as a whole entity is definitely better than testing each individual simple function. This way, the tests written should remain valid even after refactoring the source code.